AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -747,39 +758,30 @@ - (ASLayout *)layoutThatFits:(ASSizeRange)constrainedSize
 #pragma clang diagnostic pop
 }
 
+/**
+ * Calculates a new or reuses the already calculated layout based on given constrained and parent size
```

Comment:
Optional: for in-implementation comments, maybe don't need header-style comment format.
Optional: Later, maybe we should introduce a +Layout.mm category implementation file for Layout and Transition-related code.  Probably not worth doing right now, but it would be exciting to have all this important code tightly located together.